backend/src/users/users.model.ts

```diff
@@ -1,20 +1,19 @@
+export interface Lending {
+  userId: number,
+  title: string,
+  duedate: Date,
+}
+
 export interface User {
   id: number,
   email: string,
   password: string,
-  nickName: string,
-  intraId: number;
+  nickName?: string,
+  intraId?: number;
```

Comment:
intraID?: number 뒤에` , ` 가 아니라 `;`로 해도 괜찮은건가요?
```suggestion
  intraId?: number,
```

Reply:
`;` 울 `,`으로 수정하였습니다.